firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/FirebaseVertexAI.kt

```diff
@@ -43,10 +43,10 @@ class FirebaseVertex(
     )
 
   companion object {
-    val instance: FirebaseVertex
-      get() = Firebase.app[FirebaseVertex::class.java]
+    val instance: FirebaseVertexAI
+      get() = Firebase.app[FirebaseVertexAI::class.java]
   }
 }
 
-val Firebase.vertex: FirebaseVertex
-  get() = FirebaseVertex.instance
+val Firebase.vertex: FirebaseVertexAI
```

Comment:
should we change this as well?
```suggestion
val Firebase.vertexAI: FirebaseVertexAI
```